modules/switch/react/lib/Switch.tsx

```diff
@@ -21,14 +21,14 @@ const translateLength = spacing.s;
 
 const SwitchContainer = styled('div')({
   position: 'relative',
-  height: switchTapArea,
+  height: spacing.m,
```

Comment:
This is aligning `Switch`'s height to `Checkbox` so it looks more harmonious when they're on the same page.